pkgs/development/libraries/bullet/default.nix

```diff
@@ -22,26 +20,16 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs =
-    lib.optionals stdenv.hostPlatform.isLinux [
-      libGLU
-      libGL
-      libglut
-    ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      Cocoa
-      OpenGL
-    ];
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [
```

Comment:
nit: `libGL`/`libGLU`/`libglut` are aliases that resolve to the correct thing for the platform, so this conditional can be removed, which would probably improve portability to e.g. FreeBSD.